src/GitLink/LinkOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace GitLink
```

Comment:
Added.
I can even enable a StyleCop.Analyzers rule to enforce it. Although the preferred format nowadays is to _not_ use an XML header. But I'll assume you want to keep that and save a proposal to reformat for modern patterns for another time. :)